knotx-stack-manager/src/main/packaging/conf/includes/dataSourceHttp.conf

```diff
@@ -0,0 +1,23 @@
+address = ${bridge.dataSourceHttp.address}
```

Comment:
Please add more comments about the parameters.

Reply:
you can base on the comments from `knotx-stack-manager/src/main/packaging/conf/includes/serviceAdapter.conf` that were in the core